website/source/docs/agent/kv.html.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: "docs"
+page_title: "Consul KV"
+sidebar_current: "docs-agent-kv"
+description: |- 
+    Consul KV is a core feature of Consul and is installed with the Consul agent.
+---
+
+# Consul KV 
+
+Consul KV is a core feature of Consul and is installed with the Consul agent. Once installed with the agent, it will have sane defaults. Consul KV allows users to store indexed objects, its main uses are storing configuration parameters and metadata. It is a simple KV store and is not intended to be a full featured datastore, such as DynamoDB, but has some similarities. 
+
+The Consul KV datastore is located on the servers, but can be accessed by any agent (client or server). The natively integrated [RPC functionality](/docs/internals/architecture.html) allows clients for forward requests to servers, including key/value reads and writes. Part of Consul’s core design allows data to be replicated automatically across all the servers. Having a quorum of servers will decrease the risk of data loss. 
+
+## Accessing the KV store
+
+The KV store can be accessed by the [consul kv CLI subcommands](/docs/commands/kv.html), [HTTP API](/api/kv.html), and Consul UI. To restrict access, enable and configure [ACLs](https://learn.hashicorp.com/consul/advanced/day-1-operations/acl-guide). Once the ACL system has been bootstrapped, users and services, will need a valid token with KV [privileges](docs/agent/acl-rules.html#key-value-rules) to access the the data store, this includes even reads. The initial bootstrap token is a management token with unrestricted privileges and can be used, however we recommend creating a token with limited privileges. For example, you may want to create a token for an operator to use when updating objects related to the applications they manage.
```

Comment:
```
The initial bootstrap token is a management token with unrestricted privileges and 
should **not** be used for normal operations - we recommend creating a token 
with limited privileges.
```

We could recommend against using the management token a little more strongly here with language like this. If someone enables ACLs it would virtually never be "right" to use the management token for KV editing as part of day to day operations.

Reply:
@kyhavlov I completely removed any mention of the bootstrap token and updated the example to be a little more specific.